sunkit_image/coalignment/tests/test_match_template.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import pytest
+from numpy.testing import assert_allclose
+from skimage.feature import match_template
+
+import astropy.units as u
+
+from sunkit_image.coalignment.interface import REGISTERED_METHODS
+from sunkit_image.coalignment.match_template import (
+    _find_best_match_location,
+    _get_correlation_shifts,
+    _parabolic_turning_point,
+)
+
+
+def test_parabolic_turning_point():
+    assert _parabolic_turning_point(np.asarray([6.0, 2.0, 0.0])) == 1.5
```

Comment:
```suggestion
```
Do we really need unit tests for private functions? This get tested indirectly anyway.